script.js

```diff
@@ -1,121 +1,145 @@
-var input = document.querySelectorAll('input');
+var myLibrary = [];
 
-let newBook = document.querySelector('#add-new-book');
-let modal = document.querySelector('#modal');
+let library = document.getElementById('container');
+let addBookButton = document.getElementById('add-new-book');
+let form = document.getElementById('new-book');
+let modal = document.getElementById('modal');
 let overlay = document.querySelector('.overlay');
-let submitBook = document.querySelector('#submit-book');
-let container = document.querySelector('.container');
-let form = document.querySelector('form');
 
+addBookButton.addEventListener('click', toggleModal);
+overlay.addEventListener('click', toggleModal);
 
-newBook.addEventListener('click', addNewBook);
-overlay.addEventListener('click', closeModal);
-submitBook.addEventListener('click', bookDetails);
+const addBookToLibrary = (e) => {
+  e.preventDefault();
+  let bookData = [];
+  let input = document.querySelectorAll('input');
```

Comment:
Separate this into a single query for each input. Querying for all `input`s in existence can lead to problems if the HTML change. You should have a `bookTitle` input, a `bookAuthor` input, etc.